redun/file.py

```diff
@@ -710,6 +710,9 @@ def shell_copy(
         else:
             raise ValueError("At least one path must be given.")
 
+    def glob(self, pattern: str) -> List[str]:
```

Comment:
Another change tthat could be in its own PR, I think.

Reply:
I think this is ok to keep since people using k8s will likely be working with `gcs`.